prebuilt/fetch-libraries.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # Library versions
-VERSION_OpenSSL="1.0.1m"
-VERSION_Curl="7.21.1"
-VERSION_ICU="52.1"
-VERSION_CEF="39.0.2171.95"
```

Comment:
This line shouldn't have been changed (`VERSION_CEF` should still be `39.0.2171.95`)